### DISCUSSION TITLE
-

### DISCUSSION THREAD

**Discussion by gaoe6661** [Meloong-Git/PCL#2605] [提问] [answered]
啊等等，有人滥用你们的项目，不知道怎么做到的，发个issue会自动转跳到这个discuss，各位有时间可以去处理下。

https://github.com/Dimples1337/PCL2

**Comment by SuoHouDaiShi** [comment]
所以呢？是要做些什么吗？
你永远也无法改变某些↑↓做出来的事…

**Comment by iPodtouch6557** [comment]
个人认为这种事情还是眼不见为净吧……对某些人瞎掰扯没啥意义
另外，把标题改了……看一次难受一次

**Reply by gaoe6661** [reply]
抱歉大哥，我本来想改标题的，但是我改不了，好像没法改，也没法删。

**Reply by gaoe6661** [reply]
只能改内容，不能改名字，大哥

**Comment by LTCatt** [comment, marked as answer]
这个存储库我管不了，已经报过警了

**Comment by iPodtouch6557** [comment]
似乎举报有效了，那个作者所有的readme里图片的原网址都炸了

**Reply by NebulaTechs** [reply]
没爆炸,还活着

**Reply by iPodtouch6557** [reply]
是图片炸了，存储库还活着

**Reply by gaoe6661** [reply]
我这ca还能看到，是不是gfw又升级了？

**Reply by NebulaTechs** [reply]
里面的图片也没有完全爆炸

**Reply by iPodtouch6557** [reply]
存储库死了

**Reply by NebulaTechs** [reply]
终于彻底死了

**Comment by NebulaTechs** [comment]
现在这个讨论的标题怎么变了
